reference/src/taimpl/src/porting/transport.c

```diff
@@ -38,13 +39,13 @@ sa_status transport_authenticate_caller(sa_uuid* uuid) {
 
     // SecApi TA obtains the caller UUID from the underlying transport mechanism which is
     // platform dependent. The caller must be authenticated to prevent spoofing.
-    // TODO Soc Vendor: add code here to retrieve the authenticated caller UUID.
-
+    // TODO Soc Vendor: replace code here to retrieve the authenticated caller UUID. If the caller is the REE, return
```

Comment:
Why not create a function that retrieves the authenticated caller UUID and place the new function in the porting layer for the SOC manager to define? 

In that spirit could all code required to be modified by SOC vendors be abstracted into a set of functions that are defined in the porting directory, isolating all SOC vendor changes to a specific layer/directory (porting) and eliminating the need for them to modify code in the internal directory?

Reply:
I recommend adding these additional details to the README.md in the reference directory. The reason for my comment is to try and simplify the porting interface that the SOC vendor has to maintain to a single directory. As additional SOC vendors adopt SecAPI, we should ensure the process is as clear and painless as possible to reduce porting time and ensure a consistent and secure implementation.

Reply:
Done